extensions/PhabBugz/lib/Util.pm

```diff
@@ -332,4 +332,284 @@ sub set_phab_user {
   return Bugzilla->set_user($user, scope_guard => 1);
 }
 
+sub set_reviewer_rotation {
+  my ($revision) = @_;
+
+  INFO('D' . $revision->id . ': Setting reviewer rotation');
+
+  # Load a fresh version of the revision with Heralds changes.
+  $revision = Bugzilla::Extension::PhabBugz::Revision->new_from_query(
+    {phids => [$revision->phid]});
+
+  # Map of phids to blocking status
+  my $is_blocking = {};
+
+# Find out what the reviewer groups (if any) are. If the revision has a reviewer
+# group set, normally it ends in "-reviewer-rotation". Normally Herald will set this
+# if certain conditions are met. If there are no review rotation groups, then do nothing.
+  my @review_projects = get_review_rotation_projects($revision, $is_blocking);
+
+  if (!@review_projects) {
+    INFO('Reviewer rotation projects not found. Returning.');
+    return;
+  }
+
+  # After project reviewers, we also need to get a list of individual reviewers
+  my @review_users = get_review_users($revision, $is_blocking);
+
+  # If the revision is part of a stack, grab a list of current reviewers
+  # and if one of the project members is a reviewer, then assign the same reviewer
+  my @stack_reviewers = get_stack_reviewers($revision, $is_blocking);
+
+  # Once the reviewer rotation groups are determined, query Phabricator for
+  # list of group members for each and sort them by user ID descending.
+PROJECT: foreach my $project (@review_projects) {
+    INFO('Processing review project: ' . $project->name);
+
+    # Sort the project members so we always the same order as best we can.
+    my @project_members = sort { $a->id <=> $b->id } @{$project->members};
+
+    # If there are no members then we can't do anything and should just skip
+    if (!@project_members) {
+      INFO('No project members found for ' . $project->name . '. Skipping.');
+      next;
+    }
+
+    INFO('Sorted project members found: '
+        . (join ', ', map { $_->name } @project_members));
+
+    # Find the last selected reviewer for the rotation group, if one exists.
+    my $last_reviewer_phid = find_last_reviewer_phid($project);
+    INFO('Last reviewer phid found: '
+        . ($last_reviewer_phid ? $last_reviewer_phid : 'None'));
+
+    # If a last reviewer exists, we want to rotate the list to where the
+    # last reviewer is at the front of the list while maintaining order.
+    # move them to the front of the member list
+    @project_members = rotate_reviewer_list(\@project_members, $last_reviewer_phid);
+
+    # If there is only one member, then guess what? You are always getting it.
+    my $found_reviewer = 0;
```

Comment:
You can remove this variable and all references to it. All we do is set it here and then check `!$found_reviewer`, which is always True.